src/Reflection/ReflectionFunctionAbstract.php

```diff
@@ -360,7 +362,11 @@ private function nodeIsOrContainsYield(Node $node): bool
         foreach ($node->getSubNodeNames() as $nodeName) {
             $nodeProperty = $node->$nodeName;
 
-            if ($nodeProperty instanceof Node && $this->nodeIsOrContainsYield($nodeProperty)) {
+            if ($nodeProperty instanceof Node &&
```

Comment:
Operator precedence making this a bit of a puzzle to read: let's add some parentheses, please 😁